pkg/apis/pipeline/v1/result_validation_test.go

```diff
@@ -123,3 +155,129 @@ func TestResultsValidateError(t *testing.T) {
 		})
 	}
 }
+
+func TestResultsValidateValueError(t *testing.T) {
```

Comment:
nit: maybe move this test after `TestResultsValidateValue`

Reply:
done